From 59e38f6883963b96f1b4c9a807181dc2b3c61993 Mon Sep 17 00:00:00 2001
From: Sergio Pena <sergio.pena@cloudera.com>
Date: Thu, 9 Mar 2017 12:36:47 -0600
Subject: [PATCH 1027/1164] =?UTF-8?q?CDH-50483:=20Revert=20"HIVE-15952:=20Ad?=
 =?UTF-8?q?d=20blobstore=20integration=20test=20for=20CREATE=20LIKE=20(Juan=20?=
 =?UTF-8?q?Rodr=C3=ADguez=20Hortal=C3=A1,=20reviewed=20by=20Sergio=20Pena,=20?=
 =?UTF-8?q?Illya=20Yalovyy)"?=
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

This reverts commit 47d06fe0064536e194cb35d5d23f0572a398059d.

(cherry picked from commit f695feb8f4a5a76acc0e57e8d75fc0263568f0d3)

Change-Id: If420a8a16d150f43edffc8e4f33a03ba504489a2
---
 files/test_data                                    |    3 -
 .../src/test/queries/clientpositive/create_like.q  |   30 ----
 .../test/results/clientpositive/create_like.q.out  |  189 --------------------
 3 files changed, 222 deletions(-)
 delete mode 100644 files/test_data
 delete mode 100644 hive-blobstore/src/test/queries/clientpositive/create_like.q
 delete mode 100644 hive-blobstore/src/test/results/clientpositive/create_like.q.out

diff --git a/files/test_data b/files/test_data
deleted file mode 100644
index 3cc82df..0000000
--- a/files/test_data
+++ /dev/null
@@ -1,3 +0,0 @@
-1 23 a1
-2 24 b1
-3 25 c1
diff --git a/hive-blobstore/src/test/queries/clientpositive/create_like.q b/hive-blobstore/src/test/queries/clientpositive/create_like.q
deleted file mode 100644
index 38f384e..0000000
--- a/hive-blobstore/src/test/queries/clientpositive/create_like.q
+++ /dev/null
@@ -1,30 +0,0 @@
--- Create external table like should not result in data loss upon dropping the table
-DROP TABLE blobstore_partitioned_source_table;
-CREATE TABLE blobstore_partitioned_source_table (
-  a int, b int, c string
-)
-PARTITIONED BY (dt int, hour int)
-ROW FORMAT DELIMITED
-FIELDS TERMINATED BY ' '
-COLLECTION ITEMS TERMINATED BY '\t'
-LOCATION '${hiveconf:test.blobstore.path.unique}/create_like/data' ;
-LOAD DATA LOCAL INPATH '../../data/files/test_data' OVERWRITE INTO TABLE blobstore_partitioned_source_table
-PARTITION (dt=20110924, hour=1);
-LOAD DATA LOCAL INPATH '../../data/files/test_data' OVERWRITE INTO TABLE blobstore_partitioned_source_table
-PARTITION (dt=20110924, hour=2);
-LOAD DATA LOCAL INPATH '../../data/files/test_data' OVERWRITE INTO TABLE blobstore_partitioned_source_table
-PARTITION (dt=20110925, hour=1);
-LOAD DATA LOCAL INPATH '../../data/files/test_data' OVERWRITE INTO TABLE blobstore_partitioned_source_table
-PARTITION (dt=20110925, hour=2);
-
-DROP TABLE like_table;
-CREATE EXTERNAL TABLE like_table LIKE blobstore_partitioned_source_table LOCATION '${hiveconf:test.blobstore.path.unique}/create_like/data';
-
-MSCK REPAIR TABLE like_table;
-
-SELECT * FROM blobstore_partitioned_source_table;
-SELECT * FROM like_table;
-
-DROP TABLE like_table;
-
-SELECT * FROM blobstore_partitioned_source_table;
\ No newline at end of file
diff --git a/hive-blobstore/src/test/results/clientpositive/create_like.q.out b/hive-blobstore/src/test/results/clientpositive/create_like.q.out
deleted file mode 100644
index 0d362a7..0000000
--- a/hive-blobstore/src/test/results/clientpositive/create_like.q.out
+++ /dev/null
@@ -1,189 +0,0 @@
-PREHOOK: query: DROP TABLE blobstore_partitioned_source_table
-PREHOOK: type: DROPTABLE
-POSTHOOK: query: DROP TABLE blobstore_partitioned_source_table
-POSTHOOK: type: DROPTABLE
-PREHOOK: query: CREATE TABLE blobstore_partitioned_source_table (
-  a int, b int, c string
-)
-PARTITIONED BY (dt int, hour int)
-ROW FORMAT DELIMITED
-FIELDS TERMINATED BY ' '
-COLLECTION ITEMS TERMINATED BY '\t'
-#### A masked pattern was here ####
-PREHOOK: type: CREATETABLE
-PREHOOK: Input: ### test.blobstore.path ###/create_like/data
-PREHOOK: Output: database:default
-PREHOOK: Output: default@blobstore_partitioned_source_table
-POSTHOOK: query: CREATE TABLE blobstore_partitioned_source_table (
-  a int, b int, c string
-)
-PARTITIONED BY (dt int, hour int)
-ROW FORMAT DELIMITED
-FIELDS TERMINATED BY ' '
-COLLECTION ITEMS TERMINATED BY '\t'
-#### A masked pattern was here ####
-POSTHOOK: type: CREATETABLE
-POSTHOOK: Input: ### test.blobstore.path ###/create_like/data
-POSTHOOK: Output: database:default
-POSTHOOK: Output: default@blobstore_partitioned_source_table
-PREHOOK: query: LOAD DATA LOCAL INPATH '../../data/files/test_data' OVERWRITE INTO TABLE blobstore_partitioned_source_table
-PARTITION (dt=20110924, hour=1)
-PREHOOK: type: LOAD
-#### A masked pattern was here ####
-PREHOOK: Output: default@blobstore_partitioned_source_table
-POSTHOOK: query: LOAD DATA LOCAL INPATH '../../data/files/test_data' OVERWRITE INTO TABLE blobstore_partitioned_source_table
-PARTITION (dt=20110924, hour=1)
-POSTHOOK: type: LOAD
-#### A masked pattern was here ####
-POSTHOOK: Output: default@blobstore_partitioned_source_table
-POSTHOOK: Output: default@blobstore_partitioned_source_table@dt=20110924/hour=1
-PREHOOK: query: LOAD DATA LOCAL INPATH '../../data/files/test_data' OVERWRITE INTO TABLE blobstore_partitioned_source_table
-PARTITION (dt=20110924, hour=2)
-PREHOOK: type: LOAD
-#### A masked pattern was here ####
-PREHOOK: Output: default@blobstore_partitioned_source_table
-POSTHOOK: query: LOAD DATA LOCAL INPATH '../../data/files/test_data' OVERWRITE INTO TABLE blobstore_partitioned_source_table
-PARTITION (dt=20110924, hour=2)
-POSTHOOK: type: LOAD
-#### A masked pattern was here ####
-POSTHOOK: Output: default@blobstore_partitioned_source_table
-POSTHOOK: Output: default@blobstore_partitioned_source_table@dt=20110924/hour=2
-PREHOOK: query: LOAD DATA LOCAL INPATH '../../data/files/test_data' OVERWRITE INTO TABLE blobstore_partitioned_source_table
-PARTITION (dt=20110925, hour=1)
-PREHOOK: type: LOAD
-#### A masked pattern was here ####
-PREHOOK: Output: default@blobstore_partitioned_source_table
-POSTHOOK: query: LOAD DATA LOCAL INPATH '../../data/files/test_data' OVERWRITE INTO TABLE blobstore_partitioned_source_table
-PARTITION (dt=20110925, hour=1)
-POSTHOOK: type: LOAD
-#### A masked pattern was here ####
-POSTHOOK: Output: default@blobstore_partitioned_source_table
-POSTHOOK: Output: default@blobstore_partitioned_source_table@dt=20110925/hour=1
-PREHOOK: query: LOAD DATA LOCAL INPATH '../../data/files/test_data' OVERWRITE INTO TABLE blobstore_partitioned_source_table
-PARTITION (dt=20110925, hour=2)
-PREHOOK: type: LOAD
-#### A masked pattern was here ####
-PREHOOK: Output: default@blobstore_partitioned_source_table
-POSTHOOK: query: LOAD DATA LOCAL INPATH '../../data/files/test_data' OVERWRITE INTO TABLE blobstore_partitioned_source_table
-PARTITION (dt=20110925, hour=2)
-POSTHOOK: type: LOAD
-#### A masked pattern was here ####
-POSTHOOK: Output: default@blobstore_partitioned_source_table
-POSTHOOK: Output: default@blobstore_partitioned_source_table@dt=20110925/hour=2
-PREHOOK: query: DROP TABLE like_table
-PREHOOK: type: DROPTABLE
-POSTHOOK: query: DROP TABLE like_table
-POSTHOOK: type: DROPTABLE
-#### A masked pattern was here ####
-PREHOOK: type: CREATETABLE
-PREHOOK: Input: ### test.blobstore.path ###/create_like/data
-PREHOOK: Output: database:default
-PREHOOK: Output: default@like_table
-#### A masked pattern was here ####
-POSTHOOK: type: CREATETABLE
-POSTHOOK: Input: ### test.blobstore.path ###/create_like/data
-POSTHOOK: Output: database:default
-POSTHOOK: Output: default@like_table
-PREHOOK: query: MSCK REPAIR TABLE like_table
-PREHOOK: type: MSCK
-PREHOOK: Output: default@like_table
-POSTHOOK: query: MSCK REPAIR TABLE like_table
-POSTHOOK: type: MSCK
-POSTHOOK: Output: default@like_table
-Partitions not in metastore:	like_table:dt=20110924/hour=1	like_table:dt=20110924/hour=2	like_table:dt=20110925/hour=1	like_table:dt=20110925/hour=2
-Repair: Added partition to metastore like_table:dt=20110924/hour=1
-Repair: Added partition to metastore like_table:dt=20110924/hour=2
-Repair: Added partition to metastore like_table:dt=20110925/hour=1
-Repair: Added partition to metastore like_table:dt=20110925/hour=2
-PREHOOK: query: SELECT * FROM blobstore_partitioned_source_table
-PREHOOK: type: QUERY
-PREHOOK: Input: default@blobstore_partitioned_source_table
-PREHOOK: Input: default@blobstore_partitioned_source_table@dt=20110924/hour=1
-PREHOOK: Input: default@blobstore_partitioned_source_table@dt=20110924/hour=2
-PREHOOK: Input: default@blobstore_partitioned_source_table@dt=20110925/hour=1
-PREHOOK: Input: default@blobstore_partitioned_source_table@dt=20110925/hour=2
-#### A masked pattern was here ####
-POSTHOOK: query: SELECT * FROM blobstore_partitioned_source_table
-POSTHOOK: type: QUERY
-POSTHOOK: Input: default@blobstore_partitioned_source_table
-POSTHOOK: Input: default@blobstore_partitioned_source_table@dt=20110924/hour=1
-POSTHOOK: Input: default@blobstore_partitioned_source_table@dt=20110924/hour=2
-POSTHOOK: Input: default@blobstore_partitioned_source_table@dt=20110925/hour=1
-POSTHOOK: Input: default@blobstore_partitioned_source_table@dt=20110925/hour=2
-#### A masked pattern was here ####
-1	23	a1	20110924	1
-2	24	b1	20110924	1
-3	25	c1	20110924	1
-1	23	a1	20110924	2
-2	24	b1	20110924	2
-3	25	c1	20110924	2
-1	23	a1	20110925	1
-2	24	b1	20110925	1
-3	25	c1	20110925	1
-1	23	a1	20110925	2
-2	24	b1	20110925	2
-3	25	c1	20110925	2
-PREHOOK: query: SELECT * FROM like_table
-PREHOOK: type: QUERY
-PREHOOK: Input: default@like_table
-PREHOOK: Input: default@like_table@dt=20110924/hour=1
-PREHOOK: Input: default@like_table@dt=20110924/hour=2
-PREHOOK: Input: default@like_table@dt=20110925/hour=1
-PREHOOK: Input: default@like_table@dt=20110925/hour=2
-#### A masked pattern was here ####
-POSTHOOK: query: SELECT * FROM like_table
-POSTHOOK: type: QUERY
-POSTHOOK: Input: default@like_table
-POSTHOOK: Input: default@like_table@dt=20110924/hour=1
-POSTHOOK: Input: default@like_table@dt=20110924/hour=2
-POSTHOOK: Input: default@like_table@dt=20110925/hour=1
-POSTHOOK: Input: default@like_table@dt=20110925/hour=2
-#### A masked pattern was here ####
-1	23	a1	20110924	1
-2	24	b1	20110924	1
-3	25	c1	20110924	1
-1	23	a1	20110924	2
-2	24	b1	20110924	2
-3	25	c1	20110924	2
-1	23	a1	20110925	1
-2	24	b1	20110925	1
-3	25	c1	20110925	1
-1	23	a1	20110925	2
-2	24	b1	20110925	2
-3	25	c1	20110925	2
-PREHOOK: query: DROP TABLE like_table
-PREHOOK: type: DROPTABLE
-PREHOOK: Input: default@like_table
-PREHOOK: Output: default@like_table
-POSTHOOK: query: DROP TABLE like_table
-POSTHOOK: type: DROPTABLE
-POSTHOOK: Input: default@like_table
-POSTHOOK: Output: default@like_table
-PREHOOK: query: SELECT * FROM blobstore_partitioned_source_table
-PREHOOK: type: QUERY
-PREHOOK: Input: default@blobstore_partitioned_source_table
-PREHOOK: Input: default@blobstore_partitioned_source_table@dt=20110924/hour=1
-PREHOOK: Input: default@blobstore_partitioned_source_table@dt=20110924/hour=2
-PREHOOK: Input: default@blobstore_partitioned_source_table@dt=20110925/hour=1
-PREHOOK: Input: default@blobstore_partitioned_source_table@dt=20110925/hour=2
-#### A masked pattern was here ####
-POSTHOOK: query: SELECT * FROM blobstore_partitioned_source_table
-POSTHOOK: type: QUERY
-POSTHOOK: Input: default@blobstore_partitioned_source_table
-POSTHOOK: Input: default@blobstore_partitioned_source_table@dt=20110924/hour=1
-POSTHOOK: Input: default@blobstore_partitioned_source_table@dt=20110924/hour=2
-POSTHOOK: Input: default@blobstore_partitioned_source_table@dt=20110925/hour=1
-POSTHOOK: Input: default@blobstore_partitioned_source_table@dt=20110925/hour=2
-#### A masked pattern was here ####
-1	23	a1	20110924	1
-2	24	b1	20110924	1
-3	25	c1	20110924	1
-1	23	a1	20110924	2
-2	24	b1	20110924	2
-3	25	c1	20110924	2
-1	23	a1	20110925	1
-2	24	b1	20110925	1
-3	25	c1	20110925	1
-1	23	a1	20110925	2
-2	24	b1	20110925	2
-3	25	c1	20110925	2
-- 
1.7.9.5

